test/model/content_test.dart

```diff
@@ -615,6 +699,233 @@ class ContentExample {
+    '<p>'
+      '<span class="katex-display"><span class="katex">'
+        '<span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><semantics><mrow><mstyle mathsize="0.5em"><mrow><mi>λ</mi><mstyle mathsize="2.488em"><mi>λ</mi></mstyle></mrow></mstyle></mrow>'
+          '<annotation encoding="application/x-tex">\\tiny {\\lambda \\Huge \\lambda}</annotation></semantics></math></span>'
+        '<span class="katex-html" aria-hidden="true">'
+          '<span class="base">'
+            '<span class="strut" style="height:1.7278em;"></span>'
+            '<span class="mord sizing reset-size6 size1">'
+              '<span class="mord mathnormal">λ</span>'
+              '<span class="mord mathnormal sizing reset-size1 size11">λ</span></span></span></span></span></span>\n\n'
+      '<span class="katex-display"><span class="katex">'
+        '<span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><semantics><mrow><mstyle mathsize="2.488em"><mi>λ</mi><mstyle mathsize="2.074em"><mi>λ</mi><mstyle mathsize="1.728em"><mi>λ</mi><mstyle mathsize="1.44em"><mi>λ</mi><mstyle mathsize="1.2em"><mi>λ</mi><mstyle mathsize="1em"><mi>λ</mi><mstyle mathsize="0.9em"><mi>λ</mi><mstyle mathsize="0.8em"><mi>λ</mi><mstyle mathsize="0.7em"><mi>λ</mi><mstyle mathsize="0.5em"><mi>λ</mi></mstyle></mstyle></mstyle></mstyle></mstyle></mstyle></mstyle></mstyle></mstyle></mstyle></mrow>'
+          '<annotation encoding="application/x-tex">\\Huge \\lambda\n\\huge \\lambda\n\\LARGE \\lambda\n\\Large \\lambda\n\\large \\lambda\n\\normalsize \\lambda\n\\small \\lambda\n\\footnotesize \\lambda\n\\scriptsize \\lambda\n\\tiny \\lambda</annotation></semantics></math></span>'
+        '<span class="katex-html" aria-hidden="true">'
+          '<span class="base">'
+            '<span class="strut" style="height:1.7278em;"></span>'
+            '<span class="mord mathnormal sizing reset-size6 size11">λ</span>'
+            '<span class="mord mathnormal sizing reset-size6 size10">λ</span>'
+            '<span class="mord mathnormal sizing reset-size6 size9">λ</span>'
+            '<span class="mord mathnormal sizing reset-size6 size8">λ</span>'
+            '<span class="mord mathnormal sizing reset-size6 size7">λ</span>'
+            '<span class="mord mathnormal sizing reset-size6 size6">λ</span>'
+            '<span class="mord mathnormal sizing reset-size6 size5">λ</span>'
+            '<span class="mord mathnormal sizing reset-size6 size4">λ</span>'
+            '<span class="mord mathnormal sizing reset-size6 size3">λ</span>'
+            '<span class="mord mathnormal sizing reset-size6 size1">λ</span></span></span></span></span></p>',
+    [
+      MathBlockNode(
+        texSource: "\\tiny {\\lambda \\Huge \\lambda}",
```

Comment:
Can you write a widget test for this?

I think that'll actually be more pertinent than the parsing test (though we'll naturally have this parsing test as a consequence given the way the ContentExample system is structured, and that's fine): the important thing isn't that one layer says "0.5em" and then a layer inside it says "4.976em" so much as it is that the text winds up at the size that would be 2.488em if found at the root of the math tree.

For the widget test it may be helpful to have this example split into two examples: one for each of the two blocks in this example. That way the widget test can render only the one that it intends to inspect.